miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1052,7 +1052,7 @@ public int compare(SequencingParametersDto o1, SequencingParametersDto o2) {
     return dtoList;
   }
 
-  public static LibraryDto asDto(Library from) {
+  private static LibraryDto asDto(Library from, boolean initializeBoxables) {
```

Comment:
`includeX` seems to be idiomatic elsewhere in the file, as well as in the Sample method above; what do you think about changing `initializeBoxables` to `includeBoxables` here and in the Dilution asDto method below?

Reply:
changed